### PR TITLE
Gv final revision measures

### PR DESCRIPTION
**Pull request by giovannivolpe**
The branch gv-Final-revision-Measures and this pull request are to 

- [ ] introduce simple measures (Degree, DegreeAv and Distance)
- [ ] relative methods in Graph
- [ ] test functions
- [ ] make measures compatible with multilayer approach

### CODE REVIEW

File: braph2/graph/are_compatible.m [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function bool = are_compatible(g, m)
+% ARE_COMPATIBLE checks whether graph and measure are compatible
+%
+% BOOL = ARECOMPATIBLE(GRAPh, MEASURE) checks whether the graph GRAPH and
```
Comment (annacanal) [inline braph2/graph/are_compatible.m:4]: ARECOMPATIBLE(GRAP**H**, MEASURE)

File: braph2/graph/test_Measure.m [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    for j = 1:1:length(graph_class_list)
+        graph_class = graph_class_list{j};
+        A = rand(randi(10));
+        g = Graph.getGraph(graph_class, A);
+        m = Measure.getMeasure(measure_class, g);
+        value = m.getValue();
+        if m.is_global()
+            assert(isnumeric(value) && numel(value) == 1, ...
+                ['BRAPH:' measure_class ':getValueBroken'], ...
+                [measure_class '.getValue()  used with ' graph_class ' should return a scalar'])
```
Comment (annacanal) [inline braph2/graph/test_Measure.m:18]: If the asserts are the old ones, these tests in test_Measure should be better commented. It creates confusion later

File: braph2/graph/are_compatible.m (hunk 1 above)
Comment (giovannivolpe) [reply braph2/graph/are_compatible.m]: changed

File: braph2/graph/test_Measure.m (hunk 2 above)
Comment (giovannivolpe) [reply braph2/graph/test_Measure.m:18]: Yes, you are right: test_Measure has not been revised yet.

**Pull request merged**